src/subclass/object.rs

```diff
@@ -564,6 +564,17 @@ mod test {
         assert!(weak.upgrade().is_none());
     }
 
+    #[test]
+    fn test_create_child_object() {
+        let type_ = ChildObject::get_type();
+        let obj = Object::new(type_, &[]).unwrap();
```

Comment:
Oh also, please use expect, even in test!

Reply:
The tests in the whole file and in various other places are like this. Let's handle this in a separate PR to make it consistent

Reply:
Created https://github.com/gtk-rs/glib/issues/592 for that.